hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -158,7 +158,7 @@
         "cd {proj_root} && ./util/design/gen-otp-mmap.py --seed {seed}",
         // Save build_seed in a file and reuse that file during run phase.
         '''echo {seed} > {build_seed_file_path}; \
-           echo "echo create file {build_seed_file_path}"
+           echo 'echo create file {build_seed_file_path}'
```

Comment:
Do you think we could remove this altogether? I am not sure if it is even useful.

Reply:
yeah it seems to be just a debug printout and not strictly necessary. 

but we will likely remove the build_seed file mechanism entirely when switching to the bazel-generated OTP files, so I think it's ok to just leave it for now and deal with it once we cross that bridge...